src/WCSimDetectorConfigs.cc

```diff
@@ -463,41 +463,201 @@ void WCSimDetectorConstruction::SetNuPrismBeamTest_mPMTGeometry()
 	vessel_cyl_height = 38.*CLHEP::mm;    //option A, option B would be 277 mm
 	vessel_radius_curv = 342.*CLHEP::mm;  //needs to include the vessel thickness, as we construct from outside inwards.
 	vessel_radius = 254.*CLHEP::mm;
-	dist_pmt_vessel = 8*CLHEP::mm;      
+	dist_pmt_vessel = 2*CLHEP::mm;      
 	orientation = PERPENDICULAR;
 	mPMT_outer_material = "G4_PLEXIGLASS";
 	mPMT_inner_material = "Air";               // TODO: real air, hence update abs_length
 	mPMT_material_pmtAssembly = "SilGel";
 	mPMT_outer_material_d = 10*CLHEP::mm;
 	
 	// Radius of cone at z=reflectorHeight
-	id_reflector_height = 9.29*CLHEP::mm;        // From solidworks CAD drawings
-	id_reflector_z_offset = 6.12*CLHEP::mm;
-	id_reflector_angle = 46.5*CLHEP::deg; 
-	mPMT_pmt_openingAngle = 8.3*CLHEP::deg;
+	id_reflector_height = 6.53*CLHEP::mm;        // for a radius of 7.25mm, for hex: 5.4mm (radius of 6mm)
```

Comment:
Duplicated parameters from above. At the moment, these should be the same between WCTE and IWCD, so would be better to define a variable or class that's shared between geometries

Reply:
Need to double check but I don't know why I would have changed this..agree that they should be the same between IWCD and WCTE